Patch 2

sungod12 opened the pull request:
# Pull Request Template

## Description

-solved error by updating scholarship component due to not closing p tag

Fixes # 174

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)


## Checklist:

- [ ] My contribution follows the style guidelines of this project
- [ ] I have performed a self-review of my own contribution
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

sungod12 commented:
Check now

sungod12 commented:
There could be conflicts again,if you merge my commit after other commits due to some other changes.

shikha-16 commented:
@sungod12 hey, we had already solved this after reverting your PR and then notified you to make a new PR. did you pull the repo before opening this PR? because this issue wasn't there then.

sungod12 commented:
> @sungod12 hey, we had already solved this after reverting your PR and then notified you to make a new PR. did you pull the repo before opening this PR? because this issue wasn't there then.

You told me to make a new pr due to that p tag which is this.

sungod12 commented:
> > @sungod12 hey, we had already solved this after reverting your PR and then notified you to make a new PR. did you pull the repo before opening this PR? because this issue wasn't there then.
> 
> Wait,i didnt get it.So you solved the issue due to that p tag?

if it is already solved,then for what issue should i make a pull request?

shikha-16 commented:
> > > @sungod12 hey, we had already solved this after reverting your PR and then notified you to make a new PR. did you pull the repo before opening this PR? because this issue wasn't there then.
> > 
> > 
> > Wait,i didnt get it.So you solved the issue due to that p tag?
> 
> if it is already solved,then for what issue should i make a pull request?

for the image links colour
your previous PR for image links colour was causing a problem, so we reverted that and fixed the problem
now you should pull the repo, and make a new PR for the links colour

sungod12 commented:
> > > > @sungod12 hey, we had already solved this after reverting your PR and then notified you to make a new PR. did you pull the repo before opening this PR? because this issue wasn't there then.
> > > 
> > > 
> > > Wait,i didnt get it.So you solved the issue due to that p tag?
> > 
> > 
> > if it is already solved,then for what issue should i make a pull request?
> 
> for the image links colour your previous PR for image links colour was causing a problem, so we reverted that and fixed the problem now you should pull the repo, and make a new PR for the links colour

What was the problem and what should I write in new pull request.?
